voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -172,26 +175,51 @@ public SsmlDocumentBuilder() {
    * @return the document, never null
    */
   public @NonNull SsmlDocument build() {
-    StringBuilder buffer = new StringBuilder("<speak>");
+    return new SsmlDocument(build(true));
+  }
+
+  /**
+   * Builds a String representing this without SSML tags.
+   *
+   * @return The raw text, never null
+   */
+  public @NonNull String getRawText() {
```

Comment:
We'd need to implement an SSML parser, probably in `SsmlDocument` (since we already have a TODO to throw a ParseException if the constructor is supplied with invalid SSML).